.github/actions/send-teams-notification/action.yml

```diff
@@ -0,0 +1,72 @@
+description: Sends a teams notification
+name: Send a teams notification
+inputs:
+  webhook-url:
+    description: Microsoft Teams Incoming Webhooks URL
+    required: true
+  needs:
+    description: JSON parsed needs context
+    required: false
+    default: ""
+  dry-run:
+    description: Do not actually send the message
+    required: false
+    default: 'false'
+  raw:
+    description: JSON object to send to Microsoft Teams
+    required: false
+    default: ""
+  overwrite:
+    description: JSON like object to overwrite default message
+    required: false
+    default: >-
+      {
+          title: `${{github.actor}} initialized ${{ github.event_name}} event on ${{ github.repository }} of branch ${{ github.ref_name}}`,
+          "potentialAction": [
+              {
+                "@type": "OpenUri",
+                "name": "Repository",
+                "targets": [
+                  {
+                    "os": "default",
+                    "uri": "${{ github.server_url }}/${{ github.repository }}"
+                  }
+                ]
+              },
+              {
+                "@type": "OpenUri",
+                "name": "Build URL",
+                "targets": [
+                  {
+                    "os": "default",
+                    "uri": "${{ github.server_url }}/${{ github.repository }}/actions/runs/${{ github.run_id }}"
+                  }
+                ]
+              },
+              {
+                "@type": "OpenUri",
+                "name": "Compare",
+                "targets": [
+                  {
+                    "os": "default",
+                    "uri": "${{ github.event.compare }}"
+                  }
+                ]
+              }
+            ]
+      }
+runs:
+  using: composite
+  steps:
+    - uses: actions/checkout@v3
+    - uses: martialonline/workflow-status@v4
+      id: workflow_status_check
```

Comment:
better add a name to this step to explain which is the purpose of that, I am not getting it

Reply:
I have added a name against the step.